modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -4,49 +4,34 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="lvms-preface-sno-ran_{context}"]
-= Deploying {lvms}
+= About LVM Storage installation
 
-You can deploy {lvms} on a bare-metal or user-provisioned infrastructure cluster and configure it to dynamically provision storage for your workloads.
+You can install {lvms} on a bare-metal or user-provisioned infrastructure cluster and configure it to dynamically provision storage for your workloads.
 
-{lvms} creates a volume group using all the available unused disks and creates a single thin pool with a size of 90% of the volume group.
-The remaining 10% of the volume group is left free to enable data recovery by expanding the thin pool when required.
-You might need to manually perform such recovery.
-
-You can use persistent volume claims (PVCs) and volume snapshots provisioned by {lvms} to request storage and create volume snapshots.
-
-{lvms} configures a default overprovisioning limit of 10 to take advantage of the thin-provisioning feature.
-The total size of the volumes and volume snapshots that can be created on the clusters is 10 times the size of the thin pool.
-
-You can deploy {lvms} using one of the following:
-
-* {rh-rhacm-first}
-* {product-title} Web Console
+You can install {lvms} by using {product-title} or {rh-rhacm-first}.
 
 [WARNING]
 ====
-{lvms} only supports provisioning local storage on multi-node clusters. It does not support storage data replication mechanism across nodes. When you are using {lvms} on multi-node clusters, you must ensure storage data replication through active or passive replication mechanism to avoid a single point of failure.
+{lvms} only supports provisioning local storage on multi-node clusters. It does not support storage data replication mechanism across nodes. When you are using {lvms} on multi-node clusters, you must ensure storage data replication through active or passive replication mechanisms to avoid a single point of failure.
 ==== 
 
 [id="lvms-deployment-requirements-for-sno-ran_{context}"]
-== Requirements
-
-Before deploying {lvms} on {sno} clusters, ensure that the following requirements are met:
+== Requirements to install LVM Storage
 
-* You have installed {rh-rhacm-first} on an {product-title} cluster.
-* Every managed cluster has dedicated disks that are used to provision storage.
+Before deploying {lvms}, ensure that every managed cluster has dedicated disks that are used to provision storage.
```

Comment:
I am not sure which chapter has the info on configuring disks. Could you please point me to the relevant chapter?

Reply:
Do not have the link myself but something like that should exist already. I think this would be somewhere in the installation documentation and part of how to setup the hardware/infrastructure for the cluster correctly.

Reply:
There is a "[Storage](https://70717--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/storage-configuration)" section in the Post-installation configuration chapter.

Could not find any other relevant sections in the Installation and Post-installation documentation.